apps/projects/app/components/Panel/NewProject/NewProject.js

```diff
@@ -27,7 +29,7 @@ class NewProject extends React.Component {
       : this.props.onGithubSignIn
 
     if (this.state.started)
-      return <RepoSelector onCreateProject={this.props.onCreateProject} />
+      return <RepoSelector onCreateProject={this.props.onCreateProject} reposAlreadyAdded={this.props.reposAlreadyAdded}/>
```

Comment:
It seems you forgot an space character before closing tag

Reply:
Duly noted :)